### PR TITLE
add proper punctuation

### PR DESCRIPTION
**Pull request by rappo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 </head>
 <body>
 <div id="textContainer">
-  <h1>No</h1>
+  <h1>No.</h1>
```
Comment (corytheboyd) [inline index.html:22]: Will this change scale well? The extra overhead of adding a new character could come back to bite us later.

**Review by corytheboyd — CHANGES_REQUESTED** [review]
Please see my feedback

File: index.html (hunk 1 above)
Comment (rappo) [reply index.html:22]: Sounds like we need to add test coverage. Maybe we can recruit someone to manage the test suite for us and do some basic integration.

Comment (corytheboyd) [reply index.html:22]: I like where you are going there. I'll consult with the operations department to see what budget we have for contract work.

Comment (corytheboyd) [reply index.html:22]: On second thought, this change seems clear to me now. Going to merge!

**Review by corytheboyd — APPROVED** [review]
LGTM

**Pull request merged**